sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverClientIntegrationTest.java

```diff
@@ -0,0 +1,477 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What about cases where no messages are received in that time out period? Or if you try to receive two batches, one after the other?